src/Device/MultipleDevices.hpp

```diff
@@ -90,7 +90,7 @@ class MultipleDevices final : PortListener {
                           OperationEnvironment &env);
   void PutStandbyFrequency(RadioFrequency frequency, const TCHAR *name,
                            OperationEnvironment &env);
-  void ExchangeRadioFrequencies(OperationEnvironment &env);
+  bool ExchangeRadioFrequencies(OperationEnvironment &env);
```

Comment:
What does the return value mean? Documentation!

Reply:
Will add docs.